src/keys-api/interfaces/RegistryKey.ts

```diff
@@ -0,0 +1,22 @@
+export type RegistryKey = {
```

Comment:
I guess it's better to use a common interface for a Key rather than the RegistryKey